testsuite/src/it/java/io/quarkus/ts/startstop/QuarkusMavenPluginTest.java

```diff
@@ -96,6 +96,7 @@ public void infoTarget(TestInfo testInfo) throws IOException, InterruptedExcepti
 
     @Test
     @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/33403
+    @Disabled("https://github.com/quarkus-qe/quarkus-startstop/issues/356")
```

Comment:
Why is this releavant for 3.8 branch?

Reply:
`quarkus:update` tries to update to the latest version (3.9.2 at the time of writing). After an upgrade to 3.9.2 module `jakarta-test-minimal` can not be compiled anymore (due to resteasy renamings) and since this test usually runs first, the whole test class fails.

Reply:
@fedinskiy disabling is right choice, but I am not sure about the https://github.com/quarkus-qe/quarkus-startstop/issues/356 . Only thing it says is _Update to 3.9_. Have you considered that if `quarkus:update` always updates on the new version, then if we run tests with the 3.10 and want test 3.9, then 3.10 updates will be in place because the openrewrite recipes had been applied?

I know nothing about the test, but from your description it raises question if it is implemented correctly.

Reply:
Also yes, this test can use some improvement, here is a task for that: https://github.com/quarkus-qe/quarkus-startstop/issues/358